build/int.cloudbuild.yaml

```diff
@@ -21,143 +21,139 @@ steps:
       - 'TF_VAR_org_id=$_ORG_ID'
       - 'TF_VAR_folder_id=$_FOLDER_ID'
       - 'TF_VAR_billing_account=$_BILLING_ACCOUNT'
-  - id: create-all
+  - id: create all
```

Comment:
nit: we prefer to use updated stage names if possible `"init", "apply", "verify", "teardown"`